packages/client/src/contexts/SettingsContext.tsx

```
@@ -0,0 +1,118 @@
+  showTimer: true,
+  confirmLeave: true,
+  language: 'en',
+};
+
+const LS_KEY = 'durak_settings';
+
+function fromStorage(): GameSettings {
+  try {
+    const raw = localStorage.getItem(LS_KEY);
+    if (raw) return { ...DEFAULTS, ...JSON.parse(raw) };
+  } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)